contactsummary/js/src/contact-summary/controllers/key-dates.controller.js

```diff
@@ -69,9 +69,11 @@ define([
         .then(function (response) {
           angular.forEach(response, function (role) {
             var endDate = moment(role.end_date);
-            var isInFuture = (!endDate.isValid()) ? isDateInFuture(endDate) : true;
```

Comment:
So there was a bug? It passed "invalid" endDate to isDateInFuture?

Reply:
Yes! and it was always returning `true`, which raised the bug to increase the count of active job roles even for the past inactive job roles.